anthos-vmruntime/README.md

```diff
@@ -0,0 +1,405 @@
+
+---
+### Deploy an Anthos on bare metal cluster
+
+> **Note:** If you already have an Anthos on bare metal cluster then you can
+> skip this step. If you do skip, then note that some future steps also might
+> not be relavant to your setup.
+
+You can deploy an Anthos on barel metal cluster by following
+[this guide](/anthos-bm-gcp-terraform/docs/manuallb_install.md). This sets up an
+Anthos on bare metal cluster inside GCE VMs using the [`Manual LB` mode](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/installing/manual-lb). We use [Google Cloud Loadbalancers](https://cloud.google.com/load-balancing/docs/load-balancing-overview) as the manually configured loadbalancers to ensure we
+can reach the VM workloads running inside our cluster at the end of this guide.
+
+The steps that follow assumes that you have SSH'ed into the admin workstation
+that was created during the Anthos on bare metal cluster installation process.
+They also assume that you have configured the `KUBECONFIG` environment variable
+inside the admin workstation so that you can access the control plane of your
+cluster using `kubectl`.
+
+```sh
+# these are example commands from the installation guide
+
+# SSH into the admin workstation
+gcloud compute ssh tfadmin@cluster1-abm-ws0-001 --project=<YOUR_PROJECT> --zone=<YOUR_ZONE>
+
+# set the KUBECONFIG environment variable
+export KUBECONFIG=/home/tfadmin/bmctl-workspace/cluster1/cluster1-kubeconfig
+```
+---
+### Cleanup the resources created by default in the installation guide
+
+```sh
+# delete the sample application pods in the default namespace
+kubectl delete pods --all -n default
+
+# delete the sample application services in the default namespace
+kubectl delete svc api-server-svc inventory-svc payments-svc
+```
+---
+### Deploy a `MySQL` container that the VM workload will connect to
+
+Copy the [mysql-db.yaml](mysql-db.yaml) file into the admin workstation and
+apply it to the cluster.
+
+```sh
+kubectl apply -f mysql-db.yaml
+```
+---
+### Enable the `Anthos VMRuntime` on the cluster
+
+The `Anthos VMRuntime` custom resource definition (CRD) is part of all Anthos on
+bare metal clusters since version `1.10`. An instance of the `VMRuntime` custom
+resource is already created upon installation. However, it is `disabled` by
+default. Thus, you have to `enable` it to be able to create and manage VMs using
+the `Anthos VMRuntime`.
+
+The following command would open the resource specification for the `VMRuntime`
+instance in the default editor of your shell session. Update the specification
+by setting the `spec.enabled` field to `true`.
+```sh
+kubectl edit vmruntime
+```
+ You can also copy the [vmruntime.yaml](vmruntime.yaml)
+file from this repository into your admin workstation and directly apply that
+using `kubectl apply -f vmruntime.yaml`.
+
+> **Note:** Optionally, you can set the `spec.useEmulation` field to `true`.
+> Setting this to true ensures that the VMRuntime makes use of hardware
```

Comment:
@Shabirmean - I believe that setting to `true` actually permits the use of *software* emulation.  While hardware emulation should always be attempted first, if this is `false` then *software* emulation is NOT permitted.

https://github.com/GoogleCloudPlatform/anthos-samples/blame/kubevirt-guide/anthos-vmruntime/README.md#L96

Reply:
Updated in https://github.com/GoogleCloudPlatform/anthos-samples/pull/255/commits/1b21d191f2b3ddeeb42f2853b6d895307af9010e